### PR TITLE
✨ sets up table component

### PR DESCRIPTION
**Pull request by hlminh2000**

### CODE REVIEW

File: client/uikit/Table/index.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+import reactTableDefaultStyle from './reactTableDefaultStyle';
+import Typography from '../Typography';
+import ascending from '../assets/table/ascending.svg';
+import descending from '../assets/table/descending.svg';
+import unsorted from '../assets/table/unsorted.svg';
+import TablePagination from './TablePagination';
+import { useTheme } from '../ThemeProvider';
+
+const StyledTable = styled(ReactTable)`
```
Comment (joneubank) [inline client/uikit/Table/index.js]: Do you want to move this to a styledComponents.js file like with other components?

File: client/uikit/Table/index.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+export default Table;
+
+export const SelectTable = props => {
+  const { isSelected, data, keyField } = props;
+  const selectedIds = data.map(data => data[keyField]).filter(isSelected);
+  const Component = selectTable(Table);
+  return <Component {...props} isSelectTable primaryKey={keyField} selectedIds={selectedIds} />;
+};
+SelectTable.propTypes = {
+  ...SelectTable.propTypes,
```
Comment (joneubank) [inline client/uikit/Table/index.js]: Should this be ReactTable.propTypes ? This is the only place SelectTable propTypes are defined so as written this looks unnecessary.

File: client/uikit/theme/defaultTheme/table.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import colors from "./colors";
+
+export default {
+  th: {
```
Comment (joneubank) [inline client/uikit/theme/defaultTheme/table.js]: Should this be a human readable name, not the html tag name?

**Comment by joneubank** [comment]
Looks awesome. A couple small things to consider but nothing blocking.

File: client/uikit/Table/index.js (hunk 2 above)
Comment (hlminh2000) [reply client/uikit/Table/index.js]: great catch!

**Pull request merged**